aptos-move/block-executor/src/executor.rs

```diff
@@ -572,45 +565,41 @@ where
                                 finalized_groups.push((group_key, metadata_op, finalized_group));
                             },
                             Err(err) => {
-                                maybe_err = Some(err);
+                                maybe_code_err = Some(err.into());
                                 break;
                             },
                         }
-                        if maybe_err.is_some() {
+                        if maybe_code_err.is_some() {
                             break;
                         }
                     }
                 }
             }
 
-            last_input_output.record_finalized_group(txn_idx, finalized_groups);
+            // We return an error (leads to halting the execution) in the following cases:
```

Comment:
Update: I had to read the code in order to understand this comment :) Which means that the comments should probably be modified.

Don't want to be annoying - but plz don't reuse comments - reuse code :)

Reply:
where did I re-use the the comment? also please suggest how to change it, like what was confusing and happy to fix it.